components/model/VEntities.vue

```diff
@@ -23,9 +23,15 @@ export interface VEntitiesProps {
     required: boolean;
     schema: any;
     save: boolean;
+    newValueFunction?: () => any;
```

Comment:
mettre `defaultNewValue` 
```suggestion
    defaultNewValue?: (() => any) | any;
```